src/api.js

```diff
@@ -0,0 +1,61 @@
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+async function createGame() {
+  // Check if you have already created a game before
```

Comment:
- [ ] Kindly try to use the arrow functions instead of the function keyword. Arrow functions provide a simpler and more concise way to define functions. Kindly change the functions to arrow functions.